NETWORK/NetworkGetNetworkIdFromEntity.md

```diff
@@ -27,11 +28,12 @@ local netId = NetworkGetNetworkIdFromEntity(obj)
 ```
 
 ```js  
-const obj = CreateObject(GetHashKey("prop_weed_01"), vector3(0.0, 0.0, 0.0), true)
-const netId = NetworkGetNetworkIdFromEntity(obj)
+let [x, y, z] = GetEntityCoords(GetPlayerPed(-1));
```

Comment:
It's preached to use `PlayerPedId()` over `GetPlayerPed(-1)`

Reply:
Yeah, looks way cleaner that way 👍